src/azul/indexer/__init__.py

```diff
@@ -353,6 +351,12 @@ def _aggregate(self, contributions: List[Contribution]) -> List[Aggregate]:
                                   num_contributions=tallies[entity])
             aggregates.append(aggregate)
 
+        log.info('Selected %i contribution(s) to be aggregated. Breakdown by entity: %s',
```

Comment:
Maybe you want to consider moving this to line No. 343, before the aggregation happens such that 
"... to be aggregated" is factual. Or perhaps wording it in a way that indicates that the aggregation has happened.

Reply:
Good point. I think having the message logged before the action is performed makes sense. Fixup incoming.